src/Compilers/Core/Portable/Syntax/LineDirectiveMap.cs

```diff
@@ -19,7 +20,7 @@ namespace Microsoft.CodeAnalysis
     internal abstract partial class LineDirectiveMap<TDirective>
         where TDirective : SyntaxNode
     {
-        protected readonly LineMappingEntry[] Entries;
+        protected readonly ImmutableArray<LineMappingEntry> Entries;
```

Comment:
>ImmutableArray<LineMappingEntry> Entries; [](start = 27, length = 41)

This change creates too much churn to an escrow level change. Consider moving it to a separate PR targeting post RTM branch.